rllib/connectors/tests/test_agent.py

```diff
@@ -503,11 +503,12 @@ def test_connector_pipline_with_view_requirement(self):
             )
             action = policy_output[0][0]
 
-            next_obs, rewards, dones, info = env.step(action)
+            next_obs, rewards, dones, truncateds, info = env.step(action)
             env_out_dict = {
                 SampleBatch.NEXT_OBS: next_obs,
                 SampleBatch.REWARDS: rewards,
                 SampleBatch.DONES: dones,
```

Comment:
This will introduce more breaking changes through the code, but I think it's wise to change DONES -> terminated here. Changing this after the fact will only be a harder task and introduce more points for confusion.